Makefile

```diff
@@ -8,9 +8,21 @@ COMMIT_HASH := $(shell git rev-parse --short HEAD)
 LD_FLAGS = -X github.com/dymensionxyz/dymint/version.DymintGitCommitHash=$(COMMIT_HASH)
 BUILD_FLAGS = -mod=readonly -ldflags "$(LD_FLAGS)"
 CGO_ENABLED ?= 0
+BRANCH := $(shell git rev-parse --abbrev-ref HEAD)
```

Comment:
check this simplified method
https://github.com/dymensionxyz/dymension/pull/321/files